robotiq_2f_gripper_action_server/CMakeLists.txt

```diff
@@ -4,65 +4,66 @@ project(robotiq_2f_gripper_action_server)
 find_package(catkin REQUIRED COMPONENTS
   actionlib
   actionlib_msgs
+  control_msgs
   message_generation
   robotiq_2f_gripper_control
   roscpp
-  control_msgs
 )
 
 catkin_package(
   INCLUDE_DIRS include
-  CATKIN_DEPENDS actionlib actionlib_msgs robotiq_2f_gripper_control roscpp
+  CATKIN_DEPENDS actionlib actionlib_msgs control_msgs robotiq_2f_gripper_control roscpp
 )
 
 include_directories(include)
 include_directories(
   ${catkin_INCLUDE_DIRS}
-  ${robotiq_2f_gripper_action_server_INCLUDE_DIRS}
+  ${${PROJECT_NAME}_INCLUDE_DIRS}
 )
 
 # The action server
-add_executable(robotiq_2f_gripper_action_server_node
-    src/robotiq_2f_gripper_action_server_node.cpp
-    src/robotiq_2f_gripper_action_server.cpp
-    include/robotiq_2f_gripper_action_server/robotiq_2f_gripper_action_server.h)
+add_executable(${PROJECT_NAME}_node
+  include/${PROJECT_NAME}/${PROJECT_NAME}.h
+  src/${PROJECT_NAME}.cpp
+  src/${PROJECT_NAME}_node.cpp
+)
```

Comment:
I would have probably left these alone, but if `catkin_lint` likes this better ..